### PR TITLE
Configure our app using juxt's aero

### PR DESCRIPTION
**Pull request by jstepien**

### CODE REVIEW

File: src/spacy/config.clj [hunk 1]
```diff
@@ -0,0 +1,10 @@
+(ns spacy.config
+  (:require [aero.core :as aero]
+            [clojure.java.io :as io]))
+
+(defn config [profile]
+  {:pre [(contains? #{:dev :prod} profile)]}
+  (aero/read-config (io/resource "config.edn") {:profile profile}))
```
Comment (joyheron) [inline src/spacy/config.clj:7]: In the future (when we have two separate configuration options), will we read from two separate files for `dev` and `prod`?

Comment (jstepien) [reply src/spacy/config.clj:7]: We can either have two separate files or use the following approach:

```clojure
{:webserver
  {:port #profile {:default 8000
                   :dev 8001
                   :test 8002}}}
```

See also https://github.com/juxt/aero#profile

Comment (joyheron) [reply src/spacy/config.clj:7]: Ah... interesting. Ok. Looks good 😄

**Review by joyheron — APPROVED** [review]
This looks like a decent option to me.

**Pull request merged**